src/main/frontend/handler/export.cljs

```diff
@@ -219,3 +223,40 @@
       (.setAttribute anchor "href" data-str)
       (.setAttribute anchor "download" (file-name (str repo "_roam") :json))
       (.click anchor))))
+
+(defn backup-db-graph
+  [repo]
+  (when (and repo (= repo (state/get-current-repo)))
+    (when-let [backup-folder (ldb/get-key-value (db/get-db repo) :logseq.kv/graph-backup-folder)]
+      (p/let [handle (idb/get-item (str "file-handle/" backup-folder))
+              repo-name (sqlite-common-db/sanitize-db-name repo)]
+        (if handle
+          (p/let [graph-dir-handle (.getDirectoryHandle handle repo-name #js {:create true})
```

Comment:
Should we make each backup unique or replace the same one each time? Currently we're doing the latter and if we want that we should probably mention it in the description so people know only the last backup is available

Reply:
I've updated it to keep the latest 12 backups.